.changeset/plenty-rockets-breathe.md

```diff
@@ -0,0 +1,5 @@
+---
+"@khanacademy/wonder-blocks-popover": patch
+---
+
+Testing adding aria-describedby
```

Comment:
Update this to state you're adding the aria-describedby, and not testing anymore 😉